operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -53,16 +53,17 @@ type CentralSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Customizations,order=6,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Customize *CustomizeSpec `json:"customize,omitempty"`
 
+	// Deprecated field. This field will be removed in a future release.
```

Comment:
```suggestion
	// Deprecated field. This field will be removed in a future API version.
```

I don't think we can ever remove fields from an existing API version...